src/example-types/esTwoLevelAggregation.js

```diff
@@ -72,7 +76,15 @@ module.exports = {
                 key: bucket.key || key,
                 doc_count: bucket.doc_count,
               },
-              bucket.twoLevelAgg
+              bucket.twoLevelAgg ||
+                _.flow(
+                  _.mapKeys(key =>
+                    _.startsWith('twoLevelAgg_', key)
```

Comment:
what happened to the simpler version I posted which used `_.replace`?

Reply:
For `value_count` aggregation i.e.  `twoLevelAgg_value_count` it will return `value_count` but we expect `count`